Mathlib/RepresentationTheory/Maschke.lean

```diff
@@ -108,24 +107,22 @@ theorem sumOfConjugatesEquivariant_apply (v : W) :
 
 section
 
-variable [Invertible (Fintype.card G : k)]
-
 /-- We construct our `k[G]`-linear retraction of `i` as
 $$ \frac{1}{|G|} \sum_{g \in G} g⁻¹ • π(g • -). $$
 -/
 def equivariantProjection : W →ₗ[MonoidAlgebra k G] V :=
-  ⅟(Fintype.card G : k) • π.sumOfConjugatesEquivariant G
+  Ring.inverse (Fintype.card G : k) • π.sumOfConjugatesEquivariant G
```

Comment:
I'd argue that using `Invertible` was appropriate here, and that really only `exists_leftInverse_of_injective` should have been simplified to `IsUnit`; but also it doesn't matter enough here to be worth reverting.